server/aws-lsp-codewhisperer/src/language-server/netTransform/models.ts

```diff
@@ -10,6 +10,7 @@ export interface StartTransformRequest extends ExecuteCommandParams {
     SolutionConfigPaths: string[]
     ProjectMetadata: TransformProjectMetadata[]
     TransformNetStandardProjects: boolean
```

Comment:
none blocking, there could be more config value in the future, maybe consider to have a settings section in the req.json to host all config